nilearn/datasets/__init__.py

```diff
@@ -78,6 +80,7 @@
            'load_mni152_wm_mask', 'fetch_icbm152_brain_gm_mask',
            'fetch_atlas_surf_destrieux', 'fetch_atlas_talairach',
            'get_data_dirs',
+           'load_sample_motor_activation_image',
```

Comment:
You'll also need to add this to `doc/modules/datasets.rst` such that it gets properly added to the docs.

Reply:
Ah I missed that, thanks!